### PR TITLE
ENH: Add capabilities for changing UI - refactor

### PR DESCRIPTION
**Pull request by labdala**
@thewtex @PaulHax

### CODE REVIEW

**Comment by labdala** [comment]
Adds changes requested by @thewtex in #528

File: src/createViewer.js [hunk 1]
```diff
@@ -198,6 +198,18 @@ const createViewer = async (
...
+    } else if (context.uiMachineOptions === 'pydata-sphinx') {
+      options.ui = {
+        href:
+          'https://cdn.jsdelivr.net/npm/itk-viewer-bootstrap-ui@0/dist/itk-viewer-bootstrap-ui.es.js',
+        export: 'default',
+      }
+    } else if (context.uiMachineOptions === 'mui') {
+      options.ui = {
+        href:
+          'https://www.jsdelivr.com/package/npm/itk-viewer-material-ui@0/dist/materialUIMachineOptions.js.es.js',
```
Comment (thewtex) [inline src/createViewer.js:210]: `www` -> `cdn`, etc.

File: src/createViewer.js [hunk 2]
```diff
@@ -198,6 +198,18 @@ const createViewer = async (
       } else {
         options.ui = loadedUIMachineOptions.default
       }
+    } else if (context.uiMachineOptions === 'pydata-sphinx') {
+      options.ui = {
+        href:
+          'https://cdn.jsdelivr.net/npm/itk-viewer-bootstrap-ui@0/dist/itk-viewer-bootstrap-ui.es.js',
```
Comment (thewtex) [inline src/createViewer.js:204]: `itk-viewer-bootstrap-ui` -> new name

**Review by thewtex — CHANGES_REQUESTED** [review]

**Review by thewtex — APPROVED** [review]

**Comment by thewtex** [comment]
:sparkler:

**Pull request merged**